cmd/bucket-handlers.go

```diff
@@ -603,7 +603,7 @@ func (api objectAPIHandlers) PostPolicyBucketHandler(w http.ResponseWriter, r *h
 			Bucket:   objInfo.Bucket,
 			Key:      objInfo.Name,
 			ETag:     `"` + objInfo.ETag + `"`,
-			Location: getObjectLocation(objInfo.Bucket, objInfo.Name),
+			Location: getObjectLocation(globalMinioAddr, objInfo.Bucket, objInfo.Name),
```

Comment:
globalMinioAddr gets set to `:9000` if no --address is passed

Reply:
Correct.. this is waiting on a different architectural problem see above.